### DISCUSSION TITLE
Option for payout threshold for Zergpool

### DISCUSSION THREAD

**Discussion by doomnie7** [RainbowMiner/RainbowMiner#1669] [Q&A] [answered]
Maybe I missed it, but I could not find anything in the documentation on how to increase your payment threshold for Zergpool. I have look at the pools and coins config files, but could not find anything relevant.
Below is a copy from Zergpool.com with the instruction how to impliment this with direct mining.
Any help how to do this with Rainbow Miner please?

_> Personalized payment level
> Specify payment threshold using pl= parameter in password field (example pl=0.1). Setting is saved after miner is connected at least 20 mins. Personalized payment threshold level can be only higher than pool default. If you want to reset it, specify pl=0 and wait for at least 20 mins. You can check your setting was applied using your wallet address web page. ("pl" stands for payment level)
> Example to mine in Innova coin and get payout in LTC setting payment level 0.5 LTC
> -o stratum+tcp://neoscrypt.mine.zergpool.com:PORT -u <YOUR_LTC_ADDRESS> -p c=LTC,mc=INN,pl=0.5,ID=WorkerName1_

**Comment by RainbowMiner** [comment, marked as answer]
You use `"...-Params": "pl=0.5",` in pools.config.txt for this. These are the possible parameters:

**` "<WALLETCOINSYMBOL>-Params": "",`**
**` "<WALLETCOINSYMBOL>-<MININGCOINSYMBOL>-Params": "",`**

Examples in case your payout coin is LTC (and mining coin is INN, second example):
- if you want to have all algorithms/mining coins running with that "pl=0.5", you add the following in section "ZergPool" (or "ZergPoolCoins")
   `"LTC-Params": "pl=0.5",` 
- if you want to run only one specific mining coin with that "pl=0.5", you add the following in section "ZergPool" (or "ZergPoolCoins")
  `"LTC-INN-Params": "pl=0.5",`

With this extra parameter, any other parameter(s) can be added to the miner's "-p" for a specific pool/coin. Whatever you put into that field will be comma concatenated to the "-p" automatically.

Here is a more detailed example:
```
  "ZergPoolCoins": {
    "LTC": "$LTC",
    "LTC-Params": "pl=0.5",
    "Worker": "$WorkerName",
    "Penalty": "12",
...
```

**Comment by doomnie7** [comment]
Thank you very much!
